ApplicationLibCode/ProjectDataModel/RimPlotCurveAppearance.cpp

```diff
@@ -106,6 +106,13 @@ RimPlotCurveAppearance::RimPlotCurveAppearance()
                        "Minimum pixel distance between symbols",
                        "" );
 
+    CAF_PDM_InitField( &m_curveFittingTolerance,
+                       "m_curveFittingTolerance",
```

Comment:
Unusual name to be used in xml (second parameter)? Should be "CurveFittingTolerance"?